docs/locales/ko/LC_MESSAGES/manager/user-api/vfolders.po

```diff
@@ -44,21 +47,23 @@ msgid ""
 "permissions: read-only, read-write, read-write-delete. They are represented "
 "by short strings, ``'ro'``, ``'rw'``, ``'wd'``, respectively. The owner of a "
 "virtual folder have read-write-delete permission for the folder."
-msgstr ""
+msgstr "또한, 가상 폴더를 다른 사용자와 공유할 수 있으며, "
+"적절한 권한을 부여하여 초대할 수 있습니다. 현재, 권한은 각각 세 가지 레벨로 구분됩니다: 읽기 전용, 읽기-쓰기, 읽기-쓰기-삭제. "
+"이들은 각각 ``'ro'`` , ``'rw'``, ``'wd'`` 라는 짧은 문자열로 표현됩니다. 가상 폴더의 소유자는 폴더에 대해 읽기-쓰기-삭제 권한을 가집니다."
 
 #: ../../manager/user-api/vfolders.rst:21 5e48b61ced6240b7806ba6ac59f82870
 msgid "Listing Virtual Folders"
-msgstr ""
+msgstr "가상 폴더 목록"
 
 #: ../../manager/user-api/vfolders.rst:23 9d5d7ad2cb2d40bdbb92dded519d12c4
 msgid "Returns the list of virtual folders created by the current keypair."
-msgstr ""
+msgstr "현재 키쌍으로 생성된 가상 폴더의 목록을 반환합니다."
```

Comment:
Keypair is commonly used as proper noun on technical documents, so it'll be better to let it as-is.

```suggestion
msgstr "현재 키페어로 생성된 가상 폴더의 목록을 반환합니다."
```